tests/system/action/meeting/test_clone.py

```diff
@@ -530,3 +531,71 @@ def test_permissions_missing_payload_committee_permission(self) -> None:
             "Missing CommitteeManagementLevel: can_manage for committee 2",
             response.json["message"],
         )
+
+    def test_clone_with_created_topic_and_agenda_type(self) -> None:
+        self.set_models(self.test_models)
+        response = self.request(
```

Comment:
If you just want to execute an action to use the result to test another action (here, you are not interested in the result or functionality of `topic.create` but onlywant to create the topic to use it later), you can use `execute_action`. This should be much faster since it skips a lot of checks like permission etc. Also the test code becomes (hopefully) clearer because all `request` calls are actually for the tested action.

Reply:
done